index.html

```diff
@@ -0,0 +1,276 @@
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="#">Partners</a>
+      </li>
+      <li class="sec-nav-item-btn">
+        <button class="sec-nav-link-btn px-4">Campaign</button>
+      </li>
+    </ul>
+  </div>
+
+        <div id="header">
+          <div id="header-nav-bar">
+            <nav id="burguer">
+              <div id="mobile-menu">
+                <img id= "menu-burguer" src="./capstoneimgs/burguer.svg" alt="" class="menu-icon p-3">
+                <img id= "close-menu" src="./capstoneimgs/x-button-icon-20.jpg" alt="" class="close-menu-icon p-3">
+              </div>
+              <div id="menu-container">
+                <ul class="menu">
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center text-danger" >
+                    <a href="about.html">About</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center"> 
+                    <a href="#program">Program</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#join">Join</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#sponsor">Sponsor</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#news">News</a>
+                  </li>
+              </ul>
+              </div> 
+            </nav>
+          </div>                     
+        </div>
+  </section>
+
+  <main>
+        <section class="hero">
+          <div class="intro sm d-flex-column">
+            <p class="hero-hello">"Hello! World Music Fans!"</p>
+            <h1 class="hero-title">INTI RAYMI MUSIC FESTIVAL 2023</h1>
+          </div>
+        </section>
+        <section id="artists-lineup-cont">
+            <div class="artists-lineup sm d-flex row p-5 justify-content-center">
+              <p class="line1 d-flex">TAWA</p>
+              <p class="line2 d-flex">TETRÁFONO ENSAMBLE // CHANGUITAS</p>
+              <p class="line3 d-flex">TAMBORES Y OTROS DEMONIOS</p>
+              <p class="line4 d-flex">SAMY // GRUPO SENDERO // ALEX ALVEAR</p>
+              <p class="line5 d-flex">IRAIZ OVIEDO Y SU MISTURA // MARIA TEJADA</p>
+              <p class="line6 d-flex">TALLER LA BOLA // JATARI</p>
+              <p class="line7 d-flex">WAÑUKTA TONIC // KARINA CLAVIJO </p>
+              <p class="line8 d-flex">JAYAC // DANIEL MANCERO // PITEKUS</p>
+            </div>
+         
+          <div class="date-venue">
+            <p class="date mx-4">2023.06.22(THU)</p>
+            <p class="venue mx-4">@Ciudad Mitad del Mundo, Central Plaza</p>
+        </div>
+      </section>
+        
+  <section class="info">
+    <div class="title">
+        <h1 class="mp-title d-flex justify-content-center">Main Program</h1>
```

Comment:
Kindly use h1 element only once in web page. h1 tag is defines and hightlights tho most important element on the page. For here, hierarchically h2 would be more suitable.

Reply:
done!